app/cmd/cli/root.go

```diff
@@ -129,7 +145,7 @@ func start(cmd *cobra.Command, args []string) {
 		}
 		message := fmt.Sprintf("Exit signal %s received\n", sig)
 		fmt.Println(message)
-		os.Exit(3)
+		os.Exit(0)
```

Comment:
When we are sending the `os.Exit(3)`, the main process interprets child process exit as an error. (This result to the cmod.Wait() returning an error line 113).
Don't think it really fits in that scenario as we are properly shutting down the service. wdyt?